pydrad/parse/util.py

```diff
@@ -234,3 +235,48 @@ def read_hstate_file(filename):
         format='ascii',
         names=columns,
     )
+
+def read_scl_file(filename):
+    """
+    Parse the ``.scl`` files containing the time-scales as a function of position
+    """
+    columns = [
+        'coordinate',
+        'grid_widths',
+        'advective_timescale',
+        'electron_conductive_timescale',
+        'ion_conductive_timescale',
+        'viscous_timescale',
+        'collisional_timescale',
+        'radiative_timescale',
+        'free_bound_timescale',
+    ]
+    units = {
+        'coordinate': 'cm',
+        'grid_widths': 'cm',
+        'advective_timescale': 's',
+        'electron_conductive_timescale': 's',
+        'ion_conductive_timescale': 's',
+        'viscous_timescale': 's',
+        'collisional_timescale': 's',
+        'radiative_timescale': 's',
+        'free_bound_timescale': 's',
+    }
+    table = astropy.table.QTable.read(
+        filename,
+        format='ascii',
+    )
+
+    # NOTE: This is done after creating the table because the
+    # number of columns is variable depending on whether
+    # non-equilibrium ionization was switched on in the simulation.
```

Comment:
Could you add a comment about which columns the NEI option add? I'm assuming this works because the last few columns are missing in cases where the NEI option is not used.

Reply:
One extra column at the end.  Will do!  Also, thanks for reviewing before I requested.  I'm still shoring up some tests.

Reply:
I was feeling a bit proactive on the metro 🚄